dev-util/SPIRV-Headers/SPIRV-Headers-1.3.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils vcs-snapshot
+
+SNAPSHOT_COMMIT="02ffc719aa9f9c1dce5ce05743fb1afe6cbf17ea"
+SRC_URI="https://github.com/KhronosGroup/SPIRV-Headers/archive/${SNAPSHOT_COMMIT}.tar.gz -> ${P}.tar.gz"
```

Comment:
Please reorder the variables to match the skeleton:

```
DESCRIPTION
HOMEPAGE
SRC_URI

LICENSE
SLOT
KEYWORDS
IUSE
```

Reply:
fixed in all packages